lib/core/test/unit/Cardano/Wallet/Primitive/ModelSpec.hs

```diff
@@ -325,6 +347,30 @@ spec = do
         it "discovery on blocks = discovery on summary" $
             property prop_discoverFromBlockData
 
+    parallel $ describe "applyBlocks" $ do
+
+        describe "filteredTxs" $ do
+            it "prop_applyBlocks_filteredTxs_someOurs" $
+                prop_applyBlocks_filteredTxs_someOurs
+                    & property
+            it "prop_applyBlocks_filteredTxs_allOurs" $
+                prop_applyBlocks_filteredTxs_allOurs
+                    & property
+            it "prop_applyBlocks_filteredTxs_noneOurs" $
+                prop_applyBlocks_filteredTxs_noneOurs
+                    & property
+
+        describe "lastUTxO" $ do
+            it "prop_applyBlocks_lastUTxO_someOurs" $
+                prop_applyBlocks_lastUTxO_someOurs
+                    & property
+            it "prop_applyBlocks_lastUTxO_allOurs" $
+                prop_applyBlocks_lastUTxO_allOurs
+                    & property
+            it "prop_applyBlocks_lastUTxO_noneOurs" $
+                prop_applyBlocks_lastUTxO_noneOurs
+                    & property
```

Comment:
```suggestion
            it "prop_applyBlocks_filteredTxs_someOurs" $
                property prop_applyBlocks_filteredTxs_someOurs
            it "prop_applyBlocks_filteredTxs_allOurs" $
                property prop_applyBlocks_filteredTxs_allOurs
            it "prop_applyBlocks_filteredTxs_noneOurs" $
                property prop_applyBlocks_filteredTxs_noneOurs

        describe "lastUTxO" $ do
            it "prop_applyBlocks_lastUTxO_someOurs" $
                property prop_applyBlocks_lastUTxO_someOurs
            it "prop_applyBlocks_lastUTxO_allOurs" $
                property prop_applyBlocks_lastUTxO_allOurs
            it "prop_applyBlocks_lastUTxO_noneOurs" $
                property prop_applyBlocks_lastUTxO_noneOurs
```

Reply:
@Unisay Thanks for your suggestion!

I'd prefer to keep the current style, as:
- it does not violate our [Coding Standards](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md);
- we already have a precedent for using this style in many other spec modules;
- it's currently easy to see the correspondence between property name and the string identifier (see https://github.com/input-output-hk/cardano-wallet/pull/3483#discussion_r974847186);
- it's currently easy to insert/delete modifiers post hoc, for example:
    ```patch
                it "prop_applyBlocks_lastUTxO_someOurs" $
                    prop_applyBlocks_lastUTxO_someOurs
    +                   & verboseShrinking
    +                   & withMaxSuccess 1_000
    +                   & within 1_000_000
                        & property
    ```